Week-5/calculator.test.js

```diff
@@ -0,0 +1,14 @@
+const mathOperations = require('./calculator')
+
```

Comment:
Can write a test for checking existence of these three function like this:

`test("Existence check", () => {
    expect(mathOperations.sum).not.toBeNull();
  });`